lib/tls.js

```diff
@@ -33,7 +33,7 @@ const binding = process.binding('crypto');
 const Buffer = require('buffer').Buffer;
```

Comment:
This should also use destructuring.

Reply:
Thanks, I missed that one. I'll update shortly.